### PR TITLE
Pingdom endpoint

### PR DESCRIPTION
**Pull request by bmcorser**
TODO:
 - [ ] ~add token auth~
 - [x] test for view

### CODE REVIEW

**Comment by uktitools** [comment]
Can one of the admins verify this patch?

File: korben/bau/webserver.py [hunk 1]
```diff
@@ -31,7 +31,9 @@ def get_app(overrides=None):
     app_cfg.add_route('update', '/update/{django_tablename}/')
     app_cfg.add_route('get', '/get/{django_tablename}/{ident}/')
     app_cfg.add_route('validate-credentials', '/auth/validate-credentials/')
+    app_cfg.add_route('status', '/status/pingdom/')
```
Comment (sdonk) [inline korben/bau/webserver.py]: `/ping.xml`

Comment (bmcorser) [inline korben/bau/webserver.py]: :+1:

**Comment by sdonk** [comment]
LGTM as well, but CI is failing

**Comment by bmcorser** [comment]
Working locally ... debugging now

**Pull request merged**